Signal/src/call/OutboundCallInitiator.swift

```diff
@@ -45,10 +45,7 @@ import SignalMessaging
         isVideo: Bool) -> Bool {
         // Rather than an init-assigned dependency property, we access `callUIAdapter` via Environment
         // because it can change after app launch due to user settings
-        guard let callUIAdapter = SignalApp.shared().callUIAdapter else {
-            owsFail("\(TAG) can't initiate call because callUIAdapter is nil")
-            return false
-        }
+        let callUIAdapter = SignalApp.shared().callUIAdapter
```

Comment:
I'm cleaning up nullability in `SignalApp`.

Reply:
:+1: